ts/components/wallet/PaymentMethodsList.tsx

```diff
@@ -39,198 +30,61 @@ export type IPaymentMethod = Readonly<{
-    image: require("../../../img/wallet/payment-methods/satispay-logoi.png"),
-    implemented: false
+    justifyContent: "space-between",
+    flex: 1
   },
-  {
-    name: I18n.t("wallet.methods.postepay.name"),
-    image: require("../../../img/wallet/payment-methods/postepay-logo.png"),
-    implemented: false
+  row: {
+    flexDirection: "row",
+    justifyContent: "space-between"
   },
-  {
-    name: I18n.t("wallet.methods.paypal.name"),
-    image: require("../../../img/wallet/payment-methods/paypal-logo.png"),
-    implemented: false
-  },
-  {
-    name: I18n.t("wallet.methods.bank.name"),
-    image: require("../../../img/wallet/payment-methods/bancomatpay-logo.png"),
-    implemented: false
-  }
-].sort((a, b) =>
-  a.name.toLocaleLowerCase().localeCompare(b.name.toLocaleLowerCase())
-);
-
-const AddMethodStyle = StyleSheet.create({
-  transactionText: {
-    color: color(variables.colorWhite).darken(0.35).string()
-  },
-  notImplementedBadge: {
-    height: 18,
-    marginTop: 2,
-    backgroundColor: variables.lightGray
-  },
-  newImplementedBadge: {
-    height: 18,
-    marginTop: 2,
-    backgroundColor: variables.brandHighLighter
-  },
-  notImplementedText: {
-    fontSize: 10,
-    lineHeight: Platform.OS === "ios" ? 14 : 16,
-    color: customVariables.colorWhite
-  },
-  centeredContents: {
-    alignItems: "center"
-  }
+  descriptionPadding: { paddingRight: 24 }
 });
 
-class PaymentMethodsList extends React.Component<Props, never> {
-  private showHelp = () => {
-    this.props.showModal(
-      <ContextualHelp
-        onClose={this.props.hideModal}
-        title={I18n.t("wallet.whyAFee.title")}
-        body={() => <Markdown>{I18n.t("wallet.whyAFee.text")}</Markdown>}
-      />
-    );
-  };
-
-  public render(): React.ReactNode {
-    const methods: ReadonlyArray<IPaymentMethod> = [
-      {
-        ...creditCard,
-        onPress: this.props.navigateToAddCreditCard
-      },
-      ...(this.props.paymentMethods ?? []),
-      ...paymentMethods
-    ];
-    return (
-      <View>
-        <View spacer={true} large={true} />
-        <FlatList
-          removeClippedSubviews={false}
-          data={methods}
-          keyExtractor={item => item.name}
-          renderItem={itemInfo => {
-            const isItemDisabled = !itemInfo.item.implemented;
-            const disabledStyle = isItemDisabled ? styles.disabled : {};
-            const isItemNew = itemInfo.item.implemented && itemInfo.item.isNew;
-            return (
-              <ListItem
-                style={styles.listItem}
-                onPress={itemInfo.item.onPress}
-                underlayColor={underlayColor}
+const PaymentMethodsList: React.FunctionComponent<Props> = (props: Props) => (
+  <>
+    <View spacer={true} large={true} />
+    <FlatList
+      removeClippedSubviews={false}
+      data={props.paymentMethods}
+      keyExtractor={item => item.name}
+      ListFooterComponent={<View spacer />}
+      renderItem={itemInfo =>
```

Comment:
this PR lgtm
I guess it's an idea to create a dedicated function to render the item?
Could it be a good refactoring?